src/components/SelectNetworkAndAccount.svelte

```diff
@@ -47,61 +50,54 @@
       try {
         networkErrorMsg = '';
         controlKeysErrorMsg = '';
-        accountsStore.clear();
-        await getApi(network.endpoint?.toString() ?? '', thisDotApi, wsProvider);
-        await fetchAccountsForNetwork(network, thisWeb3Enable, thisWeb3Accounts, thisDotApi.api as ApiPromise);
-        await updateConnectionStatus(thisDotApi.api as ApiPromise);
+        const curNetworkApi = await createApi(network.endpoint?.toString() ?? '', wsProvider);
```

Comment:
Need to keep track of this variable and when we create a new API, first we clear the old one.